docs/variables.md

```diff
@@ -50,6 +45,9 @@ For instructions on using variable substitutions see the relevant section of [th
 | `params.<param name>` | The value of the parameter at runtime. |
 | `params['<param name>']` | (see above) |
 | `params["<param name>"]` | (see above) |
+| `params.<param name>[*]` | Get the whole param array or object.|
```

Comment:
Do we plan to group object and array types (newly supported ones) under alpha features?

Reply:
yes, better to group them or update the description